docs/AWS_Integration_Guide/Configuring_AWS_integration.md

```diff
@@ -4,7 +4,7 @@ Follow this section to integrate your AWS services with Coiote DM.
 
 ## Prerequisites
 
-- An active AWS subscription.
+- An active AWS subscription with access to IoT Core, CloudFormation and CloudWatch.
```

Comment:
Pytanie, czy nie chcielibyśmy gdzieś wypisać dokładniejszych permsów.
Poza tym, tutaj jeszcze chyba trzeba wpisać dostęp do lambd i secretsmanagera, bo z nich korzystamy

Reply:
Właśnie nie jestem pewien czy chcemy wrzucać takie jsony do dev-zone'a. A z labmd i secrets managera sam użytkownik bezpośrednio nie korzysta, nie musi tam wchodzić

Reply:
Ale chodzi o sam access, nie wiem jak to wygląda, ale zakładam, że skrypt cfn nie zrobi więcej niż może użytkownik, więc jeśli nie ma dostępu do np. lambdy to się stack nie postawi